libdevice/imf_utils/double_convert.cpp

```diff
@@ -441,4 +441,13 @@ DEVICE_EXTERN_C_INLINE
 double __devicelib_imf_ull2double_rz(unsigned long long int x) {
   return __ull2double_rz(x);
 }
+
```

Comment:
I was just curious why only the _rte version is provided for this conversion and _rtp, _rtn and _rtz are not provided.

Thanks

Reply:
Hi, @asudarsa 
This PR aims to provide some util functions to do type cast between half and other types, the motivation is CUDA math provides such functionalities while SYCL doesn't, providing those utils can help users who need to port CUDA sources to SYCL. In CUDA math APIs, NV only provides rte for double to half, this is whey we only provide rte here.

Thanks very much.